### PR TITLE
feat: add design system's variables in project

### PR DESCRIPTION
**Pull request by Howard-Cui**

### CODE REVIEW

File: src/pages/font.module.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
+@use "@/styles/designSystem.scss" as var;
+
+
+.test{
```
Comment (tatoMa) [inline src/pages/font.module.scss:4]: So when out team member needs to use the text style you have defined here, they also need to wrap the text style classes inside a .text container?

Comment (Howard-Cui) [reply src/pages/font.module.scss:4]: yes

File: src/pages/font.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import Head from "next/head";
+import classes from "./font.module.scss";
+
+export default function fome() {
```
Comment (Prom1s1ngYoung) [inline src/pages/font.tsx:4]: Naming issue here. Component function name should be upper camel case.

Comment (Prom1s1ngYoung) [reply src/pages/font.tsx:4]: Also, the name of the file "font.tsx" is confusing.

**Review by HjsCS — APPROVED** [review]

**Pull request merged**